app/controllers/returns/credits/ConvertedCreditsWeightController.scala

```diff
@@ -70,8 +69,7 @@ class ConvertedCreditsWeightController @Inject()(
     val userAnswersSaveFunc = cacheConnector.saveUserAnswerFunc(request.pptReference)
     request.userAnswers.changeWithPath(userAnswerPath, answer, userAnswersSaveFunc)
       .map { _ =>
-        val claimedCredits = ClaimedCredits(request.userAnswers)
-        val nextPage = navigator.convertedCreditsWeight(mode, claimedCredits)
+        val nextPage = controllers.returns.credits.routes.ConfirmPackagingCreditController.onPageLoad(mode)
```

Comment:
shouldn't this call the navigator to get the next page?

Reply:
is it worth it? This will just have a call to the controllers.returns.credits.routes.ConfirmPackagingCreditController.onPageLoad(mode). I know this is the pattern we usually adopt, but the navigator would have not logic at all. I was thinking that may was not
 worth to inject the navigator and add one more dependency that we do not need.